nixio/data_array.py

```diff
@@ -141,6 +142,22 @@ def append_range_dimension(self, ticks, label=None, unit=None):
             self.force_updated_at()
         return rdim
 
+    def append_data_frame_dimension(self, data_frame, column=None):
+        """
```

Comment:
Can you specify the arguments in the docstrings, In particular the meaning and caveats of providing a column or not. Also column is an index, right?

Reply:
Good point.  Perhaps `column_index`, although a bit long, would be a more self-explanatory argument name.